### PR TITLE
Change data so less identifiable information in create claim draft - …

### PR DESCRIPTION
**Pull request by pragnesh-patel**
Need to get this in before Automation review

### CODE REVIEW

File: src/main/app/drafts/draft-data/claimDraft.ts [hunk 1]
```diff
@@ -11,37 +11,37 @@ export function prepareClaimDraft (userEmailAddress: string) {
...
-            day: 27
+            year: 2000,
+            month: 1,
+            day: 1
           }
         }
       },
       mobilePhone: {
-        number: '07112358132'
+        number: '(0)207 127 0000'
```
Comment (timja) [inline src/main/app/drafts/draft-data/claimDraft.ts]: isn't this not a valid mobile?

Comment (jacmurphy) [reply src/main/app/drafts/draft-data/claimDraft.ts]: I asked this to pragnesh at the time and he said it was fine

Comment (timja) [reply src/main/app/drafts/draft-data/claimDraft.ts]: has it been tested? I don't think it is fine

Comment (pragnesh-patel) [reply src/main/app/drafts/draft-data/claimDraft.ts]: @timja @jacmurphy - Its asking a claimant for a telephone number - The field is free text and was changed to that, so I think its valid to add a telephone number -  I can refactor it, but I just did the minimum change here

Comment (timja) [reply src/main/app/drafts/draft-data/claimDraft.ts]: actually you're right 👍

**Pull request merged**